templates/nexus-default.properties

```diff
@@ -0,0 +1,10 @@
+# Jetty section
```

Comment:
Could we keep the "lineinfile" method instead of overriding the whole file via templating (see https://github.com/savoirfairelinux/ansible-nexus3-oss/blob/master/tasks/nexus_install.yml#L118 )  ? Future Nexus releases may add some other mandatory default properties here.